### PR TITLE
Added several language specific meetups

### PR DESCRIPTION
**Pull request by timonv**
Just a few, plenty more!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,11 @@ Want to see more awesomeness? [Follow us on Twitter](https://twitter.com/madeawk
 - [Makersclub](http://makersclubams.com/) - Bimonthly casual drinkup without an agenda.
 - [Hackers & Founders](https://www.meetup.com/Hackers-and-Founders-Amsterdam-NL/) - Monthly meetup at Cafe De Doffer.
 - [The Next Web Conference](https://thenextweb.com/conference) - Biggest tech conference (May 18 & 19).
-- [AMSxTech](http://amsxtech.com/) - Community driven conference (March 31).
+- [AMSxTech](http://amsxtech.com/) - Community driven conference (March 31) and monthly meetups.
 - [FunctionConf](https://functionconf.io/) - Conference about serverless architectures (June 15 & 16).
+- [Amsterdam-rb](https://www.meetup.com/amsterdam-rb/) - Active Ruby meetup with frequent, large events
```
Comment (davidvanleeuwen) [inline README.md]: Please follow the [contribution guidelines](https://github.com/awkward/awesome-events/blob/master/contributing.md) and end all descriptions with a full stop/period.

**Review by davidvanleeuwen — CHANGES_REQUESTED** [review]
Thanks for your submission, but could you change the following:

**Comment by timonv** [comment]
Updated!

**Review by davidvanleeuwen — APPROVED** [review]
Thanks 👍

**Pull request merged**